modules/channel-messenger/src/renderers/dropdown.ts

```diff
@@ -0,0 +1,32 @@
+import * as sdk from 'botpress/sdk'
+import { ChannelRenderer } from 'common/channel'
+import { CHANNEL_NAME } from '../backend/constants'
+import { MessengerContext } from '../backend/typings'
+
+export class MessengerDropdownRenderer implements ChannelRenderer<MessengerContext> {
+  get channel(): string {
+    return CHANNEL_NAME
+  }
+
+  get priority(): number {
+    return -1
+  }
+
+  get id(): string {
+    return MessengerDropdownRenderer.name
+  }
+
+  handles(context: MessengerContext): boolean {
+    return !!context.payload.options?.length
+  }
+
+  render(context: MessengerContext) {
+    const payload = context.payload // as sdk.DropdownContent
```

Comment:
Why not add the `sdk.DropdownContent` with this PR ?

Reply:
As I mentionned in the description we'll think about the wider support for dropdowns later. This PR just makes sure dropdowns work at the moment